### ISSUE TITLE
BlockAdBlock - Mod_PageSpeed Wont Work Together 

### ISSUE THREAD

**Issue by ghost** [apache/incubator-pagespeed-mod#1416]
Hey GitHub so i have been trying to use BlockAdBlock and Mod_PageSeed together and it wont work i have no clue why can someone please find a fix or something i used the stable and the beta and it still wont work for both of them

**Comment by jeffkaufman** [comment]
Could you say more about how they're not working together?

**Comment by ghost** [comment]
the adblock clode is already minified and google somehow minifies the sht out of it and it dosen't work it remove detection code

**Comment by jmarantz** [comment]
Most important: what version of mod_pagespeed are you using?

Which filters do you have enabled?

I took a quick look at https://blockadblock.com/ .  Is that the tool you
are using?  So the JS you've injected is in
https://blockadblock.com/blockadblock_basic_script.php ?

If it's really our minifier that is causing the problem, you can prevent it
from running on inline scripts with:
  ModPagespeedDisableFilter rewrite_javascript_inline
mod_pagespeed will still rewrite the external JS in that case.

Please let us know if that helps.  If you've found a bug in our minifier we
definitely want to know.

**Comment by ghost** [comment]
Hello,
Version: X-Mod-Pagespeed: 1.11.33.4-0 
Yea thats the one but its a customized script - - - http://pastebin.com/15gYyGpa
The Problem is definitely cause by the minifer i disabled it and it worked like a charm

<>Sorry for the response delay i had to take a trip outside of US, internet was sht.

**Comment by jmarantz** [comment]
I took a look through this JS and I think what's happening is that, in an attempt to obscure itself from AdBlockers, the script embeds meaningful content in comments. JS Comments are visible to JS, but our minifier strips them anyway.

I don't think there's anything to fix in the minifier, and the workaround I suggested should stand as the solution for now.

There's another bug open suggesting we provide a mechanism to disable rewriters from HTML attributes, and that would be a slightly more targeted workaround, described in 

https://github.com/pagespeed/mod_pagespeed/issues/1320

I'm going to close this bug and leave that one to track the issue.
